extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveProcessor.java

```diff
@@ -814,23 +814,15 @@ public void securityExceptionMappers(BuildProducer<ExceptionMapperBuildItem> exc
         exceptionMapperBuildItemBuildProducer.produce(new ExceptionMapperBuildItem(
                 AuthenticationCompletionExceptionMapper.class.getName(),
                 AuthenticationCompletionException.class.getName(),
-                Priorities.USER, false));
-        exceptionMapperBuildItemBuildProducer.produce(new ExceptionMapperBuildItem(
-                AuthenticationFailedExceptionMapper.class.getName(),
-                AuthenticationFailedException.class.getName(),
-                Priorities.USER + 1, false));
```

Comment:
This and the entry for `UnauthorizedExceptionMapper` were removed because these 2 mappers use `@ServerExceptionMapper` and are registered in the `registerCustomExceptionMappers` method